app/views/manuals/updates.html.erb

```diff
@@ -1,40 +1,15 @@
 <% content_for :extra_headers do %>
   <meta name='description' content="List of updates to '<%= presented_manual.full_title %>'." />
 <% end %>
-
 <% content_for :title, 'Updates - ' + presented_manual.full_title %>
-<%= render partial: "header", locals: { presented_manual: presented_manual } %>
-<div class='manual-body' id="content">
-  <article aria-labelledby="section-title">
-    <h1 id="section-title" class='section-title'><span class='title'>Updates: <%= presented_manual.title %></span></h1>
-    <div class='section-content'>
-    <% presented_manual.change_notes.by_year.each do |year, updates_by_year| %>
-      <div class='js-collapsible-collection subsection-collection'>
-        <p class='section-title'><%= year %></p>
-        <div class='collapsible-subsections'>
-          <% updates_by_year.each do |day, updated_documents| %>
-            <h2>
-              <span class='subsection-title-text'>
-                <%= marked_up_date(day) %>
-                <span class="visuallyhidden">
-                  published amendments
-                </span>
-              </span>
-            </h2>
-            <dl class='updates-list'>
-            <% updated_documents.each do |document| %>
-              <dt><%= link_to document.title, document.base_path %></dt>
-                <% document.change_notes.each do |note| %>
-                  <dd>
-                    <%= note %>
-                  </dd>
-                <% end %>
-            <% end %>
-            </dl>
-          <% end %>
-        </div>
-      </div>
-    <% end %>
-    </div>
-  </article>
-</div>
+<%= render 'header', presented_manual: presented_manual %>
+<% if presented_manual.beta? %>
+  <%= render partial: 'govuk_component/beta_label' %>
+<% end %>
```

Comment:
I can see why we wouldn't put the beta-label into the header partial (because we're changing the condition for showing the beta-label in the new_navigation versions).  I still think the partial is in the wrong place though on this page.